test/contract/blackduck.e2e.test.ts

```diff
@@ -34,23 +34,9 @@ describe('Blackduck flow contract', () => {
     expect(resp).toBe(0)
   })
 
-  it('With missing mandatory fields blackduck.url', async () => {
-    mockBridgeDownloadUrlAndSynopsysBridgePath()
-    mockBlackduckParamsExcept(['BLACKDUCK_INSTALL_DIRECTORY', 'BLACKDUCK_SCAN_FAILURE_SEVERITIES', 'BLACKDUCK_URL'])
-
-    setAllMocks()
-
-    try {
-      const resp = await run()
-    } catch (err: any) {
-      expect(err.message).toContain('failed with exit code 2')
-      error(err)
-    }
-  })
-
   it('With missing mandatory fields blackduck.api.token', async () => {
     mockBridgeDownloadUrlAndSynopsysBridgePath()
-    mockBlackduckParamsExcept(['BLACKDUCK_INSTALL_DIRECTORY', 'BLACKDUCK_SCAN_FAILURE_SEVERITIES', 'BLACKDUCK_API_TOKEN'])
+    mockBlackduckParamsExcept(['BLACKDUCK_INSTALL_DIRECTORY', 'BLACKDUCK_SCAN_FAILURE_SEVERITIES', 'BLACKDCK_API_TOKEN'])
```

Comment:
BLACKDCK_API_TOKEN should be BLACKDUCK_API_TOKEN? Look at the blackduck spelling.

Reply:
good catch!

Reply:
renamed it.